core/controller/src/main/scala/org/apache/openwhisk/core/controller/BasicAuthenticationDirective.scala

```diff
@@ -39,7 +39,6 @@ object BasicAuthenticationDirective extends AuthenticationDirectiveProvider {
       Try {
         // authkey deserialization is wrapped in a try to guard against malformed values
         val authkey = BasicAuthenticationAuthKey(UUID(pw.username), Secret(pw.password))
-        logging.info(this, s"authenticate: ${authkey.uuid}")
```

Comment:
Is this to enhance security?

Reply:
There was a concern raised via direct message about this information being logged.

Reply:
I considered it leaves the user id to authenticate, not sure it would be an issue.

Reply:
A concern was raised about this logging statement by an enterprise that was evaluating openwhisk.  I agree it is just the username portion that is being logged, not the password.  However the enterprise felt that this information leakage was a security concern.  As there seems to be little value to logging the data, removing it seemed like a reasonable way to address the concern.

Reply:
ok got it.